test/functional/zapwallettxes.py

```diff
@@ -20,11 +20,9 @@
                                  )
 
 class ZapWalletTXesTest (BitcoinTestFramework):
-
-    def __init__(self):
-        super().__init__()
+    def set_test_params(self):
         self.setup_clean_chain = True
-        self.num_nodes = 2
+        self.num_nodes = 3
```

Comment:
why is this changed in a refactoring commit?

Reply:
fixed